index.Rmd

```diff
@@ -0,0 +1,181 @@
+
+```{r}
+birth_graph(age)
+```
+
+
+# Life Expectancty and Ethnicity
+
+From this graph it is clear that life expectancy has risen over the years. This was expected as modern medicine has developed and become more wide spread (another study for another time). There is a significant difference between the life expectancies of White, Black and Hispanic women, with Hispanic women having the highest life expectancy and Black women having the lowest. 
+
+```{r}
+types_of_women <- c("All Women (Age)", "White Women (Age)", "Black Women (Age)", "Hispanic Women (Age)", "White non Hispanic Women (Age)", "Black non Hispanic Women (Age)")
+
+life_double <-
+  life_expect %>% 
+  select(-c('1980', '1981', '1982', '1983', '1984', '1985', '1986', '1987', '1988', '2020')) %>% 
+  slice(-c(26:33)) %>% 
+  filter(!Years %in% c('Men (Years)', 'By gender', 'By race and ethnicity')) %>% 
+  slice(-c(1, 3, 5, 7, 9, 11)) %>% 
+  mutate(Type_of_Women = types_of_women) %>% 
+  select(-Years)
+
+life_wide <- 
+  life_double  %>% 
+  pivot_longer(!Type_of_Women, names_to = "year", values_to = "life_expectancy") %>% 
+  rename(Year = year) %>% 
+  mutate(Year = as.numeric(Year)) %>% 
+  select(Year, life_expectancy, Type_of_Women) %>% 
+  arrange(Year) %>% 
+  pivot_wider(names_from = Type_of_Women, values_from = life_expectancy)
+
+life_long <- 
+  life_double  %>% 
+  pivot_longer(!Type_of_Women, names_to = "year", values_to = "life_expectancy") %>% 
+  rename(Year = year) %>% 
+  mutate(Year = as.numeric(Year)) %>% 
+  select(Year, life_expectancy, Type_of_Women) %>% 
+  arrange(Year) %>% 
+  mutate(Type_of_Women = str_remove_all(string = Type_of_Women,
+                               pattern = "\\(Age\\)"))
+```
+
+
+```{r}
+life_graph(race_and_ethnicity)
+```
+
+The following graph shows the same decrease in births after the year 2007 that the "Mother's Age" graph showed.
+
+```{r}
+birth_graph(ethnicity)
+```
+
+
+# Life Expectancty and Age
+
+Here we look at each individual trend by age group. The graph below show that there is a clear downward trend in the total births from mothers who are 29 years old or younger.
+
+```{r}
+births_life_joined <- 
+  births_wide %>% 
+  inner_join(life_wide, by = "Year")
+```
+
+```{r}
+births_life_joined %>% 
+  select(c('Year', contains(c('1', '2')), 'All Women (Age)')) %>% 
+  pivot_longer(!Year, names_to = "Type", values_to = "births_age") %>% 
+  ggplot(aes(x = Year, y = births_age, color = Type))+
+  ylab("Number of Births / Age of Mother")+         
+  geom_point()+
+  geom_smooth()+
+  ggtitle(str_c("Births", "from", "Younger", "Mothers", "compared", "to", "Avg", "Life", "Expectancy", sep = " "))+
+  theme(plot.title = element_text(hjust = 0.5))+
+  facet_grid( Type ~ ., scales = "free_y")+
+  theme_bw()+
+  theme(strip.text = element_blank())
```

Comment:
This graph is REALLY challenging to interpret. I'm also not entirely sure what the value of the Life Expectancy graph is here. There's something to be said for demonstrating that births by younger moms are decreasing, but how does layering in life expectancy change the story?